lib/src/excel.dart

```diff
@@ -19,6 +19,13 @@ class ExcelDart {
   static const _horizontalAlignment = 'Center';
   static const _borderStyle = 'Continuous';
   static const _currencyFormat = '\$#,##0.00';
+  static const _badFont = '#9C0006';
+  static const _badFill = '#FFC7CE';
+
+  static const _spentColumn = '\$N2';
+  static const _budgetColumn = '\$M2';
+  static const _startDateColumn = '\$O2';
+  static const _endDateColumn = '\$P2';
```

Comment:
this will all change if you add something to the schema right? is there a way to calculate this value so changes to the schema will cascade without manual updates?